hack/testing/test-upgrade.sh

```diff
@@ -32,6 +32,32 @@ function dumpEvents() {
 }
 trap dumpEvents EXIT
 
+function createOldIndexPattern() {
+  echo "creating index with old pattern"
+
+  indexDate=`date +%Y.%m.%d`
+  #rawUUID="$(tr -dc 'a-f0-9' < /dev/urandom | head -c 32)"
+
+  #echo $rawUUID
+  #echo ${rawUUID:0:8}-${rawUUID:8:4}-${rawUUID:12:4}-${rawUUID:16:4}-${rawUUID:20}
+
+  #genUUID="$(echo ${rawUUID:0:8}-${rawUUID:8:4}-${rawUUID:12:4}-${rawUUID:16:4}-${rawUUID:20})"
+  genUUID="7601c01e-012e-7640-42ab-01e833d4fd33"
```

Comment:
I think the pipefail was due to this:
```
tr -dc 'a-f0-9' < /dev/urandom | head -c 32
```
When `head` stops reading from the pipe, that causes `tr` to issue a SIGPIPE